less/home.less

```diff
@@ -0,0 +1,103 @@
+.jumbo {
+  background: transparent url("../img/home/home-jumbotron.png") no-repeat center center /cover;
+  height: 598px;
+}
+.jmbo-txt {
+  .jumbo-text;
+  margin: 0 0 49px 9.5%;
+}
+.snj {
+  .sub-header-text;
+  color: @white;
+  margin: 23px 0 0 9%;
+}
+.hamburger {
+  margin: 25px 0 0 20%;
+}
+.crd-text {
+  display: flex;
+  flex-flow: column;
+  justify-content: space-around;
+  margin: 100px 0 100px 0;
+  h2 {
+    .sub-header-text;
+  }
+  button {
+    height: 50px;
+    width: 33%;
+    border: solid 1px @off-black;
+  }
+}
+
+.crd {
+  margin: 55px 0 0 1%;
+}
+
+.fancy {
+    h2 {
+      .sub-header-text
+    }
+    img {
+      margin: 50px 0;
+      .villas {
+        position: relative;
+        margin: 0;
+      }
+    }
+    
+    .wrap-villas {
+      margin-top: 50px;
+    }
+    .overlay {
```

Comment:
Great job on the overlays for the image.  Dont forget the 1px solid white borders